crates/turborepo-lib/src/run/mod.rs

```diff
@@ -170,7 +170,8 @@ impl<'a> Run<'a> {
             opts.cache_opts.skip_remote = !enabled;
         }
 
-        let _is_structured_output = opts.run_opts.graph.is_some() || opts.run_opts.dry_run_json;
+        let _is_structured_output = opts.run_opts.graph.is_some()
+            || matches!(opts.run_opts.dry_run, Some(DryRunMode::Json));
```

Comment:
Not related to this PR, but what is this used for?

Reply:
I have no clue honestly